RELEASE.md

```diff
@@ -0,0 +1,32 @@
+# Eclipse Che release process
+
+##### 1. Create branch for release preparation and next bugfixes:
+* `git branch {branchname} #e.g 7.7.x`
+* `git push --set-upstream origin {branchname}`
+##### 2. Create PR for switch master to the next development version :
+* `git branch set_next_version_in_master_{next_version} #e.g 7.8.0-SNAPSHOT`
+* Update parent version : `mvn versions:update-parent  versions:commit -DallowSnapshots=true -DparentVersion={next_version}`
+* Update dependencies: `sed -i -e "s#{version_old}#{next_version}#" pom.xml`
+* `git commit`
+* `git push --set-upstream origin set_next_version_in_master_{next_version}`
+* Create PR
```

Comment:
link to samples please. See https://github.com/eclipse/che-machine-exec/blob/master/RELEASE.md for examples of how to do this.